config/routes.rb

```diff
@@ -1,5 +1,7 @@
 Rails.application.routes.draw do
   scope "(:locale)", locale: /en|vi/ do
     root to: "static_pages#index"
+
+    resources :questions
```

Comment:
only vào bé bì 
chị nghĩ nên là 
```suggestion
    resources :subjects do
      resources :questions, only: %i(new create)
    end
```
vì question thuộc về 1 subject nhỉ
nếu ko đúng thì thôi :v